### PR TITLE
Update copyrights year in LICENSE

### PR DESCRIPTION
**Pull request by bocharsky-bw**
Is there a better way? E.g. write it as `2015+` to avoid changing it every year?

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2022 Fabien Potencier
+Copyright (c) 2015-2023 Fabien Potencier
```
Comment (GromNaN) [inline LICENSE:1]: See https://github.com/symfony/symfony/pull/49088
```suggestion
Copyright (c) 2015-present Fabien Potencier
```

Comment (bocharsky-bw) [reply LICENSE:1]: @stof sounds great! But what is the approach exactly? I see Symfony still uses  `2015-2023`, see https://github.com/symfony/symfony/blob/6.3/LICENSE#L1 - so I will keep the PR unchanged for now

**Comment by bocharsky-bw** [comment]
Failed PHP CS Fixer build fixed in #1396

File: LICENSE (hunk 1 above)
Comment (stof) [reply LICENSE:1]: @bocharsky-bw see the Symfony PR linked at the beginning of that conversation. I know it is not merged yet, but it is already approved

Comment (bocharsky-bw) [reply LICENSE:1]: Now I see, thanks!

**Comment by javiereguiluz** [comment]
Victor, thanks for fixing this issue ... and for doing it permanently 🙌

**Comment by javiereguiluz** [comment]
@bocharsky-bw while trying to merge I see this error 🤔 

```
  Error Output:
  ================
  ERROR: Permission to bocharsky-bw/demo.git denied to javiereguiluz.
  fatal: Could not read from remote repository.

  Please make sure you have the correct access rights
  and the repository exists.
```

Not sure what can I do on my side.

**Comment by bocharsky-bw** [comment]
Hm, super weird... I have "Allow edits and access to secrets by maintainers" enabled for this PR... and no special security for that repo too. Probably a temporary GitHub outage? I also squashed commits here to see if helps. If no - probably easier to recreate a PR

**Comment by javiereguiluz** [comment]
Hi Victor ... thanks a lot for checking this. Actually, I read the error message too fast. Apparently, the issue is that I no longer have permission to push to `symfony/demo` either:

```
ERROR: Permission to symfony/demo.git denied to javiereguiluz.
fatal: Could not read from remote repository.
```

I've asked @fabpot, so hopefully we can sort this out. Thanks.

**Comment by javiereguiluz** [comment]
The problem was solved, so your PR are now fully merged.

Victor, I'm sorry again for these troubles and I thank you for your patience 🙏

**Pull request merged**